src/components/Checkout/CheckoutReceipt.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+	<div class="checkout-receipt">
+		<h2 class="checkout-receipt__headline">
+			Order Confirmation
+		</h2>
+		<section class="section">
+			TODO: Date<br>
+			{{ lender.firstName }} {{ lender.lastName }}<br>
+			{{ lender.email }}
+		</section>
+		<section>
+			<ul class="checkout-receipt__loan-list">
+				<li
+					class="section"
+					v-for="item in receipt.items.values"
+					:key="item.id"
+				>
+					<div
+						class="loan"
+						v-if="item.basketItemType === 'loan_reservation'"
+					>
+						<h3>
+							<router-link
+								class="loan__name"
+								:to="`/lend/${item.loan.id}`"
+							>
+								{{ item.loan.name }}
+							</router-link>
+						</h3>
+						<div class="loan__meta">
+							<p class="loan__meta-city">
+								<span v-if="item.loan.geocode && item.loan.geocode.city">
+									{{ item.loan.geocode.city }},
+								</span>
+								<span v-if="item.loan.geocode && item.loan.geocode.country">
+									{{ item.loan.geocode.country.name }}
+								</span>
+							</p>
+							<p>{{ item.loan.use }}</p>
+						</div>
+						<div class="loan__amount">
+							${{ item.price }}
+						</div>
+					</div>
+					<div
+						class="loan"
+						v-else-if="item.basketItemType === 'kiva_card'"
+					>
+						<h3 class="loan__name">
+							Kiva Card
+						</h3>
+						<div class="loan__amount">
+							${{ item.price }}
+						</div>
+					</div>
+					<div
+						class="loan"
+						v-else-if="item.basketItemType === 'donation'"
+					>
+						<h3 class="loan__name">
+							Donation
```

Comment:
The old receipt has the donation included in the totals section, rather than as a line item https://github.com/kiva/kiva/blob/development/httpdocs/views/Basket/ThanksView.tpl#L125-L130